runtime/include/QuantumDevice.hpp

```diff
@@ -165,7 +165,7 @@ struct QuantumDevice {
      * @param state A state vector of size 2**len(wires)
      * @param wires The wire(s) the operation acts on
      */
-    virtual void SetState(DataView<std::complex<double>, 1> &state, std::vector<QubitIdType> &wires)
+    virtual void SetState(DataView<std::complex<double>, 1> &, std::vector<QubitIdType> &)
```

Comment:
Didn't know this was possible in a function definition 😅 

(Also please clean up the PR description, the first part is not meant to be posted 😅 )

Reply:
Do you mean "Context"?

Reply:
> Didn't know this was possible in a function definition 😅

Just learned that from @erick-xanadu this week 😅

Reply:
> Do you mean "Context"?

The part with the checkboxes 😄 Erick edited it already :)